org.sixtyninestudios.kublit.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=Kublit
+Comment=new game from 69 studios
+Exec=/app/bin/Kublit.sh
```

Comment:
this should NOT be an absolute path.

Reply:
Wdym?

Reply:
Just use `Exec=Kublit.sh` and let the `PATH` look it up, same as when you type `ls` in a terminal and it finds `/usr/bin/ls` for you.